tox.ini

```diff
@@ -178,3 +178,12 @@ commands =
                 --hidden-import pytest_doctestplus.plugin \
                 --hidden-import pytest_mpl.plugin
     ./run_astropy_tests --astropy-root {toxinidir}
+
+
+[flake8]
+max-line-length = 100
+exclude = extern,*parsetab.py,*lextab.py
+
+[pycodestyle]
+max-line-length = 100
+exclude = extern,*parsetab.py,*lextab.py
```

Comment:
Do we even use `pycodestyle` anymore? If not maybe, we could just drop it and move the `flake8` config to a `.flake8` file.

Reply:
Oh no. Not another file 😭.

Reply:
Flake8 puts their own config in ``tox.ini``...

Reply:
We can keep it in the `tox.ini`. It is just odd to me to stick it in a random place like the `tox.ini` as I think of that as the place the `tox` related stuff goes, not settings for other tools. (It is too bad `flake8` won't just enable/allow support for `pyproject.toml` like most other tools now)

Reply:
I know. It's very irksome. flake8 likes ini files.

Reply:
That `ruff` package you shared looks great. Let's keep an eye on it!

Reply:
> Do we even use pycodestyle anymore?

IDK. I know Flake8 wraps pycodestyles, but I've never looked too deeply at the configuration settings.

Reply:
I'll try removing the config and let's see what happens

Reply:
Guess it passes!